src/main/java/product/Product.java

```diff
@@ -1,38 +1,31 @@
 package main.java.product;
 
-import org.jetbrains.annotations.Nullable;
-
-import static main.java.product.EnumProduct.*;
-
 public class Product {
-    drink drinkFind = drink.Cola;
-    food foodFind = food.potato;
+    Drink drinkFind = Drink.COLA;
+    Food foodFind = Food.POTATO;
 
-    private @Nullable drink drink() {
+    private void Drink() {
         switch (drinkFind) {
-            case Cola -> System.out.println("コーラ");
-            case GrapefruitJuice -> System.out.println("グレープフルーツジュース" );
-            case OrangeJuice -> System.out.println("オレンジジュース");
+            case COLA -> System.out.println("コーラ");
+            case ORANGE_JUICE -> System.out.println("グレープフルーツジュース" );
+            case GRAPEFRUIT_JUICE -> System.out.println("オレンジジュース");
         }
-        return null;
-    }
-    public drink getDrinkFind() {
-        return this.drink();
     }
 
+    public void getDrinkFind() {
+        this.Drink();
+    }
 
-    private @Nullable food food() {
+    private void Food() {
```

Comment:
メソッド名は小文字スタートでお願いします！
```suggestion
    private void food() {
```

Reply:
またメソッド名がfood（食べ物）ですと何をしたいのか？が伝わりづらいです。
メソッド名については動詞で表現しましょう。

Reply:
何なら日本語で教えてくれれば良い英語お伝えします！
DeepLなど活用してもいいかと思います。

Reply:
`メソッド名　英語`で調べていろいろ参考にするのもいいかなと思います。
https://www.google.com/search?q=%E3%83%A1%E3%82%BD%E3%83%83%E3%83%89%E5%90%8D+%E8%8B%B1%E8%AA%9E&rlz=1C5CHFA_enJP991JP991&oq=%E3%83%A1%E3%82%BD%E3%83%83%E3%83%89%E5%90%8D%E3%80%80%E8%8B%B1%E8%AA%9E&aqs=chrome..69i57j0i8i30j0i5i30l2j0i8i30.5231j0j7&sourceid=chrome&ie=UTF-8

Reply:
メソッドをpublicに変更し、メソッド名の変更。不要なメソッドを削除 #6
https://github.com/watanabemasao/EnumTest/issues/6#issue-1265991352

https://github.com/watanabemasao/EnumTest/pull/1/commits/02cc25487183c03da7aa4a7e805fe9ce56e103af#diff-45db72aefbeb310ec6ee43dece3ec185b15d640ff112ca0d3b1a1e8f3846c63b
ご確認よろしくお願いします。